spec/support/shared/unit/provider/file.rb

```diff
@@ -533,7 +533,7 @@ def unlink
           provider.load_current_resource
           tempfile = double("Tempfile", path: "/tmp/foo-bar-baz")
           allow(content).to receive(:tempfile).and_return(tempfile)
-          expect(File).to receive(:exists?).with("/tmp/foo-bar-baz").and_return(true)
+          expect(File).to receive(:exist?).with("/tmp/foo-bar-baz").and_return(true)
```

Comment:
this one and the one below it smell like you may need to worry about ordering here